src/commands/environments/destroy.ts

```diff
@@ -30,6 +30,11 @@ export default class EnvironmentDestroy extends BaseCommand {
       default: false,
       sensitive: false,
     }),
+    strict: booleanString({
```

Comment:
If I remember correctly, this was also going to be hidden for now

Reply:
set flag to hidden